.golangci.yml

```diff
@@ -35,6 +35,8 @@ linters-settings:
     forbid:
       - p: ^exec\.Command.*$
         msg: use ee/allowedcmd functions instead
+      - p: ^os\.Exit.*$
```

Comment:
Nice. You could add panic as well

Reply:
Oh sure, let me take a look! I was also waffling on adding `logutil.Fatal` -- I only had to remove one and we don't have any others and I think with slogger we probably won't add more, so maybe not useful, but couldn't hurt

Reply:
We have a couple `panic`s, largely in dataflattentable, that are a little bit complicated to pull out. I am going to punt this to a follow-up PR, if that's ok!

Reply:
PR here: https://github.com/kolide/launcher/pull/1697

Based on this branch, so the diff will look cleaner after this is merged and that is rebased.